HLTrigger/special/src/HLTRegionalEcalResonanceFilter.cc

```diff
@@ -32,12 +32,14 @@ HLTRegionalEcalResonanceFilter::HLTRegionalEcalResonanceFilter(const edm::Parame
     selePtPairBarrel_region1_ = barrelSelection.getParameter<double> ("selePtPairBarrel_region1");   
     seleS4S9GammaBarrel_region1_ = barrelSelection.getParameter<double> ("seleS4S9GammaBarrel_region1");  
     seleIsoBarrel_region1_ = barrelSelection.getParameter<double> ("seleIsoBarrel_region1");  
+    seleNxtalBarrel_region1_ = barrelSelection.getParameter<int> ("seleNxtalBarrel_region1");  
```

Comment:
please use `unsigned int` or `uint32_t` for these parameters, as I don't think it makes sense to have negative values